Parsers/ASimAuthentication/Parsers/ASimAuthenticationSshd.yaml

```diff
@@ -19,6 +19,7 @@ Notes: |
   - Certain preauth events log unsuccessful authentication negotiation, which have security importance but are not authentication events per se, and therefore are not used by this parser.
   - The parser may support other sshd servers, but this has not been tested. 
 ParserName: ASimAuthenticationSshd
+EquivalentBuiltInParser: _ASim_Authentication_Sshd
```

Comment:
@KustoKing Are these changes due to positions changing, is there anything added new or got removed?
Other than EquivalentBuiltInParser property?

Reply:
Added  property